### ISSUE TITLE
.pre-commit-config.yaml Template file not found

### ISSUE THREAD

**Issue by jimsmith** [aws-cloudformation/cfn-lint#1142]
*cfn-lint version: 0.24.2* 
*pre-commit 1.18.3*

*Description of issue.*
Template file not found

Contents of `.pre-commit-config.yaml`
```
repos:
- repo: https://github.com/aws-cloudformation/cfn-python-lint
  rev: master
  hooks:
  - id: cfn-python-lint
#    entry: cfn-lint --ignore-templates .pre-commit-config.yaml -t templates/nlb.yaml
    args: [-t templates/nlb.yaml --debug]
```

When using pre-commit this does not work:

```
git add .pre-commit-config.yaml ; pre-commit
AWS CloudFormation Linter................................................Failed
hookid: cfn-python-lint

E0000 Template file not found:  templates/nlb.yaml --debug
 templates/nlb.yaml --debug:1:1
2019-09-27 08:15:27,900 - cfnlint.decode - ERROR - Template file not found:  templates/nlb.yaml --debug
```

However when I do `pip install cfn-lint`  then comment out the args line and uncomment out the entry line it then works:

```
git add .pre-commit-config.yaml ; pre-commit
AWS CloudFormation Linter................................................Failed
hookid: cfn-python-lint

W2001 Parameter VpcName not used.
templates/nlb.yaml:18:3

E3012 Property Resources/NetworkLoadBalancer/Properties/LoadBalancerAttributes/0/Value should be of type String
templates/nlb.yaml:94:10
```

And when trying this way as per your README.md https://github.com/aws-cloudformation/cfn-python-lint/blob/master/README.md#pre-commit 
```
  - id: cfn-python-lint
#    entry: cfn-lint --ignore-templates .pre-commit-config.yaml -t templates/nlb.yaml
#    args: [-t templates/nlb.yaml --debug]
    files: templates/.*\.(json|yml|yaml)$
```

This is presented back:

```
git add .pre-commit-config.yaml ; pre-commit
AWS CloudFormation Linter............................(no files to check)Skipped
```

**Comment by kddejong** [comment]
Looks like it is how the pre commit sends the command to cfn-lint.  So this worked for me.  Looks like you would just need to add commas to separate the parameters in that array.

```yaml
repos:
  - repo: https://github.com/aws-cloudformation/cfn-python-lint
    rev: master
    hooks:
      - id: cfn-python-lint
        #    entry: cfn-lint --ignore-templates .pre-commit-config.yaml -t templates/nlb.yaml
        args: [-t, templates/vpc.yaml, --debug]
```

**Comment by kddejong** [comment]
Your last one is interesting.  This works for me and my files are being checked.  I may need to know a little bit more here.  OS, Python Version?
```yaml
repos:
  - repo: https://github.com/aws-cloudformation/cfn-python-lint
    rev: master
    hooks:
      - id: cfn-python-lint
        files: templates/.*\.(json|yml|yaml)$
```
